peekaboo/toolbox/cortex.py

```diff
@@ -56,12 +57,31 @@ def __init__(self, analyzer):
 
 class CortexAnalyzerReport:
     """ Cortex analyzer report base class. """
+
+    report_schema_taxonomies = schema.Schema([{
+        "level": schema.Schema(schema.Or("info", "malicious", "safe")),
+        "namespace": schema.Schema(schema.Or("FileInfo", "VT")),
```

Comment:
If used in actual reports wouldn't this make any report accept `FileInfo` and `VT` and possibly others if extended?

Reply:
Addressed in recent commit.